src/main/java/by/exadel/internship/service/FormService.java

```diff
@@ -16,10 +22,21 @@
 
     private final FormRepository formRepository;
 
-    public void saveForm(FormRegisterDTO formRegisterDTO) {
 
+    public Form saveForm(FormRegisterDTO formRegisterDTO) {
         Form form = mapper.toFormEntity(formRegisterDTO);
-        formRepository.save(form);
+        return formRepository.save(form);
+    }
 
+    public void uploadFile(MultipartFile file) {
```

Comment:
Where do you store location of uploaded file?

Reply:
create directory "Files"

Reply:
@ShirshovAlex Under location I mean "path to file". Do you save path in the DB?

Reply:
yes i do